### PR TITLE
[UPDATE]: Add button & Adjust position of Community Partners' description

### LINKED ISSUE #187: [FEATURE] Update & Add button to apply for community partners
### Detailed description

Can this text move upward?

image

~~-~~
We will add a button for: **Become a community partner** 

I'm not sure if it will look good to put it in this section 
image

or it will look good in the bottom?

### Context

We discuss this before but as of now, no one working on this.

### Possible implementation

_No response_

### Additional information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Simperfy**
#### Description of the change
- Moved the position of description upwards
- Adjusted the spacing of UI
- Closes #187 

#### Screenshots
Current *(desktop)*:
image

Proposed changes *(desktop)*:
image

<hr>

Current *(mobile)*:
image

Proposed changes *(mobile)*:
image

#### Checklist

**I agree to the following :-**

* [x]   Added description of the change
* [x]   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
* [x]   Search previous suggestions before making a new PR, as yours may be a duplicate.
* [x]   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/188"></a>

### CODE REVIEW

File: src/pages/community-partners.astro
Comment (Simperfy) [inline src/pages/community-partners.astro:54]: The [issue](https://github.com/web3community/devprotocol.xyz/issues/187) didn't specify where the button should point to, so I leave it empty for now.

Comment (Panquesito7) [inline src/pages/community-partners.astro:54]: Actually, now that you mention the link, @Simperfy, I was thinking to make a form for community partnerships. Google Forms would be good (or forms directly on our website). @web3community/dev-protocol-devx, @web3community/core-maintainer, thoughts? I could work on this as well. 🙂

Comment (krishguptadev) [reply src/pages/community-partners.astro:54]: I think gforms is bit uncomfortable, plus this should be asked from the devprtcl team not us 😅 @aggre

Comment (vinzvinci) [reply src/pages/community-partners.astro:54]: We can use choose one of these if ever https://tailwindui.com/components/marketing/sections/contact-sections

Comment (Simperfy) [reply src/pages/community-partners.astro:54]: Looks like we need some feedback from dev protocol team, let me know if anything comes up.

Comment (vinzvinci) [reply src/pages/community-partners.astro:54]: I'm now asking aggre and mayu. Please wait for the response.

Comment (vinzvinci) [reply src/pages/community-partners.astro:54]: https://tailwindui.com/components/marketing/sections/contact-sections#component-dec976a631662b509173ba8b848c49cd we decided to choose this.

The fields information will be 

Community Name
Website
Email
Why do you want to be partner with us?
How did you hear about us?

Comment (vinzvinci) [reply src/pages/community-partners.astro:54]: I think add that feature first before we merge this, so we can direct them there if they click **Become a Community Partner** ?

Comment (Simperfy) [reply src/pages/community-partners.astro:54]: > https://tailwindui.com/components/marketing/sections/contact-sections#component-dec976a631662b509173ba8b848c49cd we decided to choose this.
> 
> The fields information will be
> 
> Community Name Website Email Why do you want to be partner with us? How did you hear about us?

Where would we store this information? Do we have a database for this?

Comment (Simperfy) [reply src/pages/community-partners.astro:54]: > I think add that feature first before we merge this, so we can direct them there if they click **Become a Community Partner** ?

Yeah sure.

**Review by Panquesito7 — APPROVED** [review]
For me, this is good 🚀 I recommend adding the form in a separate PR so that at least the prototype is ready and we can get to the next step. 😃

**Review by krishguptadev — APPROVED** [review]

**Pull request merged**
